code/modules/station_goals/bluespace_tap.dm

```diff
@@ -250,6 +168,27 @@
 		list(1, MACH_CENTER, 1),
 		list(1, 0,		   1),
 	))
+	event_list = list(
+		new /datum/bluespace_tap_event/dirty(src),
+		new /datum/bluespace_tap_event/electric_arc(src),
+		new /datum/bluespace_tap_event/electric_arc/mass(src),
+		new /datum/bluespace_tap_event/radiation(src),
+		new /datum/bluespace_tap_event/gas/nitrogen(src),
+		new /datum/bluespace_tap_event/gas/carbon_dioxide(src),
+		new /datum/bluespace_tap_event/gas/oxygen(src),
+		new /datum/bluespace_tap_event/gas/plasma(src),
+		new /datum/bluespace_tap_event/gas/agent_b(src),
+		new /datum/bluespace_tap_event/gas/sleeping_gas(src)
+	)
```

Comment:
these should be generated and deleted AS NEEDED, not kept in a list, see `/obj/machinery/atmospherics/supermatter_crystal/proc/try_events()`